app/controllers/external_link/blog_controller.rb

```diff
@@ -0,0 +1,10 @@
+module ExternalLink
+  class BlogController < ApplicationController
+    layout nil
+
+    def show
+      person = Person.find(params[:id])
+      redirect_to person.blog, allow_other_host: true
```

Comment:
Let's use redirection with 302 status code.

Reply:
Before you do this, please check what HTTP code is used in this redirection. We could have a test to check this.